mxcube3/ui/components/SampleQueue/QueueControl.js

```diff
@@ -110,27 +110,34 @@ export default class QueueControl extends React.Component {
   }
 
   render() {
-    const idx = this.props.queue.indexOf(this.props.mounted);
     let nextSample = [];
-
-    if (this.props.queue[idx + 1]) {
-      const sampleData = this.props.sampleList[this.props.queue[idx + 1]];
-      const sampleName = sampleData.sampleName ? sampleData.sampleName : '';
-      const proteinAcronym = sampleData.proteinAcronym ? `${sampleData.proteinAcronym} - ` : '';
-
-      nextSample = [{ text: `Next Sample (${proteinAcronym}${sampleName})`,
+    let idx = -1;
```

Comment:
This definition of idx seems unnecessary, cant it be moved to the line just under where idx is really set ?, idx is never
reassigned so it can be a const ...

Reply:
This is still an issue